AMW_angular/io/src/app/resources/resources-list/resources-list.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="container">
+  @if (resourceType()){
+    <h1>{{resourceType().name}}</h1>
+    <table class="table table-sm table-striped">
+      <thead>
+      <tr>
+        <th>Resource name</th>
+        <th>Release</th>
+        <th>Actions</th>
+      </tr>
+      </thead>
+      <tbody>
+        @for (resource of resourceGroupList(); track resource.id) {
+          <tr>
+            <td>{{ resource.name }}</td>
+            <td>{{ resource.defaultRelease ? resource.defaultRelease.release : '' }}</td>
+            <td>
+              <a href="/AMW_web/pages/editResourceView.xhtml?ctx=1&id={{ resource.id }}">
```

Comment:
I have a feeling that the link is not working, but I don't know why? @StephGit

Reply:
@BeriBoss 
Regarding `ResourceGroup.java > createByResource` there is a defaultResource returned. In the old UI the id of the defaultResource or first entry of resourceGroups is mapped. (maybe always the same?)

Tracedown:

ResourceListDataProvider.java -> getResourceGroupsForSelectedResourceType() -> L107
ResourceListController.java -> loadResourceGroupsForResourceType() -> L74
ResourceGroup.java -> createByResource() -> L59